spec/requests/user/user_destroy_spec.rb

```diff
@@ -36,9 +36,9 @@
         expect(controller.current_user).to eq nil
       end
 
-      it "redirects to root" do
+      it "redirects to sign up" do
         delete "/users/full_delete"
-        expect(response).to redirect_to "/"
+        expect(response).to redirect_to new_user_registration_path
         expect(flash[:global_notice]).to include("Your account deletion is scheduled")
```

Comment:
Maybe we could change this to check for the message in the response body? This would e.g. protect us if someone removes rendering of `global_notice` from the layout.

Reply:
I took a look at this. Since we're redirecting, I'm thinking that the spec would live in the new view if/when we change this.

`follow_redirect!` doesn't quite get us where we need in this spec. Is there another way that's better to handle this case? I'm going to merge this, for now, @citizen428, but I'll open a separate PR for a hot improvement if needed 👍

Reply:
I probably just didn't think this through properly, all good 😃